go/kubebuilder/memcached-operator/README.md

```diff
@@ -41,16 +41,18 @@ Build the Memcached operator image and push it to a public registry, such as qua
 ```
 $ make install
```

Comment:
It should not be part of the section building and should be performed after before the deploy. 
Note that make install and will install the CRD's in the cluster. Here we should use the make manifests instead.